### PR TITLE
Changed constants

### PR DESCRIPTION
**Pull request by TetianaParanich**
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2474/allure/

### CODE REVIEW

**Review by penfriend — APPROVED** [review]
Great job!!!

File: cypress/support/constants.js [hunk 1]
```diff
@@ -35,11 +35,26 @@ export const LOST_ITEM_FEES_POLICY_NAMES = {
...
+  ON_ORDER: 'On order',
+  IN_PROCESS: 'In process',
+  AVAILABLE: 'Available',
+  MISSING: 'Missing',
+  IN_TRANSIT: 'In transit',
+  PAGED: 'Paged',
+  AWAITING_PICKUP: 'Awaiting pickup',
+  CHECKED_OUT: 'Checked out',
+  DECLARED_LOST: 'Declared lost',
+  AWAITING_DALIVERY: 'Awaiting delivery'
```
Comment (penfriend) [inline cypress/support/constants.js:57]: AWAITING_DALIVERY -> AWAITING_DELIVERY

File: cypress/support/constants.js [hunk 2]
```diff
@@ -35,11 +35,26 @@ export const LOST_ITEM_FEES_POLICY_NAMES = {
 
 export const LOAN_TYPE_NAMES = {
   CAN_CIRCULATE: 'Can circulate',
+  COURSE_RESERVES:'Course reserves'
```
Comment (penfriend) [inline cypress/support/constants.js:38]: COURSE_RESERVES:'Course reserves' -> COURSE_RESERVES:' Course reserves'
please add space after :

File: cypress/support/constants.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/constants.js:57]: Done

File: cypress/support/constants.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/constants.js]: Done

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Pull request merged**